packages/admin/src/Pages/Actions/Concerns/CanBeOutline.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Filament\Pages\Actions\Concerns;
+
+trait CanBeOutline
```

Comment:
`CanBeOutline` sounds weird - `CanHaveOutline` or `CanBeOutlined` perhaps?

Reply:
I feel like `CanHaveOutline` sounds like the outline is in addition to the existing style, whereas it's really instead of. Shall I go with `CanBeOutlined`?